### PR TITLE
README Update

### PR DESCRIPTION
**Pull request by Kwok-he-Chu**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -6,59 +6,52 @@
...
+|              **Pay By Link** |                                                      [Link](https://github.com/adyen-examples/adyen-java-spring-online-payments/tree/main/paybylink-example)                                                      |                        [Link](https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/paybylink-example)                         |                                                     [Link](https://github.com/adyen-examples/adyen-node-online-payments/tree/main/paybylink-example)                                                     |
+
+
+| Plugins                                                                    |
```
Comment (ayodejidev) [inline profile/README.md:36]: Great work! The headless integration example is not showcased here. Is it possible to have headless or advance integration option for plugins. The Salesforce team are also working on a headless sample.

File: profile/README.md [hunk 2]
```diff
@@ -6,59 +6,52 @@
...
+|              **Pay By Link** |                                                      [Link](https://github.com/adyen-examples/adyen-java-spring-online-payments/tree/main/paybylink-example)                                                      |                        [Link](https://github.com/adyen-examples/adyen-dotnet-online-payments/tree/main/paybylink-example)                         |                                                     [Link](https://github.com/adyen-examples/adyen-node-online-payments/tree/main/paybylink-example)                                                     |
+
+
+| Plugins                                                                    | 
+|:---------------------------------------------------------------------------| 
+| [Magento 2](https://github.com/adyen-examples/adyen-magento-plugin-demo)   |
```
Comment (ayodejidev) [inline profile/README.md:38]: The logos of the plugins too will be nice to have :)

File: profile/README.md (hunk 1 above)
Comment (jlengrand) [reply profile/README.md:36]: Plugins are also online payments only correct?

Comment (ayodejidev) [reply profile/README.md:36]: Yes it is but it does not fit in the language/framework table. All the use case listed above can all be done with one plugin sample.

Comment (Kwok-he-Chu) [reply profile/README.md:36]: I've turned it into: headless and plugin-integration (for now), let me know what you think @ayodejidev

**Pull request closed without merging**